torch_frame/data/mapper.py

```diff
@@ -209,3 +209,59 @@ def forward(
 
     def backward(self, tensor: Tensor) -> pd.Series:
         raise NotImplementedError
+
+
+class TextTokenizationTensorMapper(TensorMapper):
+    r"""Tokenize any text series into a dictionary of
+    :class:`MultiNestedTensor`.
+
+    Args:
+        text_tokenizer (callable): A callable function that takes list of
+            strings and returns a dictionary of :class:`MultiNestedTensor`.
+            The values of the dictionary contains for example tokens and
+            attention masks.
+        batch_size (int, optional): The mini-batch size used for the text
```

Comment:
Just curious, why is mini-batch necessary if we are not having parallelism

Reply:
Tokenizer does not have this issue as they don't need GPU. For convenience I just keep the interface the same. @weihua916 Do you think we can remove the `batch_size` for text tokenized?

Reply:
Discussed with @weihua916 , for now just keep it same with text embedder cfg. In the future we can easily delete it.